feedback_tool.py

```diff
@@ -0,0 +1,71 @@
+import psycopg2
+import random
+import streamlit as st
+from transformers import BartForConditionalGeneration, BartTokenizer
```

Comment:
I think we should have some other module or even other process that's generating the contexts and questions; this one can just handle collecting the ratings.